### PR TITLE
a bit hacky way to fix style sheet link

### PR DESCRIPTION
**Pull request by FlyingUnicornSF**
a bit hacky way to fix style sheet link, but this should work. Added app page title and subtitle while I'm at it.  index.html line 42 had to be hardcoded since url_for by flask doesn't seem to get rid of leading hash unelss more library get imported and add some ugly things. url_for index added. Hopefully this works.

### CODE REVIEW

File: index.py [hunk 1]
```diff
@@ -8,7 +8,8 @@
 
 
 
-app.config['APPLICATION_ROOT'] = os.environ.get('ROOT_PREFIX', '/applist')
+# app.config['APPLICATION_ROOT'] = os.environ.get('ROOT_PREFIX', '/applist')
+app.config['APPLICATION_ROOT'] = '/applist'
```
Comment (kkellerlbl) [inline index.py]: This is now hardcoded?  It might break the current CI deploy.

Comment (kkellerlbl) [reply index.py]: ...I can change CI to use /applist but it might be nicer to have it customizable.

Comment (FlyingUnicornSF) [reply index.py]: ok I will put it back

Comment (kkellerlbl) [reply index.py]: ...actually I think we changed it in CI already.

**Comment by FlyingUnicornSF** [comment]
added this in readme
## Environment Variables
KBASE_ENDPOINT: https://kbase.us/services
ROOT_PREFIX: /applist

**Pull request merged**